main.go

```diff
@@ -42,6 +42,14 @@ type flagVal struct {
 	OverWrite_flag bool
 	Instance_flag  string
 	Verbose_flag   bool
+	File_flag      bool
+}
+
+// contains local and server paths
+type pathVal struct {
+	RootWorkingDirectoryLocal  string
+	RootWorkingDirectoryServer string
+	StartingPathServer         string
```

Comment:
Looking at how RootWorkingDirectoryServer is used, I realize that it is redundant and unnecessary. It is only passed to NewDownloader and within that initializer it is transformed into the same path as StartingPathServer.